apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/ChecklistUtils.tsx

```diff
@@ -93,113 +95,134 @@ export type ChecklistStatus = {
+        'The Analytics Data API allows Contentful to fetch analytics data from the property you specify. Please enable this API inside your Google Cloud project to continue.'
+      ),
       title: CHECKLIST_NAMES.dataApi,
       description: 'Analytics Data API is not yet enabled',
       disabled: false,
     },
     invalidServiceAccount: {
       icon: getClockIcon(
-        'Service account errors detected, please install a correctly configured service account installation'
+        'This check will not run until a valid service account key has been provided'
       ),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Awaiting a correctly configured service account installation',
+      description: `Provide a valid service account key to run this check`,
       disabled: true,
     },
     error: {
-      icon: getErrorIcon('Failed to connect to the Data API - did you enable the api?'),
+      icon: getErrorIcon(
+        'The Analytics Data API allows Contentful to fetch analytics data from the property you specify. Please enable this API inside your Google Cloud project.'
+      ),
       title: CHECKLIST_NAMES.dataApi,
-      description: 'Analytics Data API must be enabled to use this app',
+      description: 'Analytics Data API is not enabled',
       disabled: false,
     },
   },
   GA4Properties: {
     success: {
-      icon: getSuccessIcon(
-        'Your service account has access to assign a Google Analytics 4 property'
-      ),
+      icon: getSuccessIcon(''),
       title: CHECKLIST_NAMES.ga4Properties,
-      description: 'Success!',
+      description: 'Service account has "viewer" access',
       disabled: false,
       checklistUrl: {
         title: 'Details',
         url: 'https://analytics.google.com/analytics/web/',
       },
     },
     firstTimeSetup: {
-      icon: getClockIcon(`Check will run once the ${CHECKLIST_NAMES.adminApi} is enabled`),
+      icon: getArrowIcon(
+        `You need to grant viewer access to your service account in a Google Analytics 4 property.`
+      ),
+      title: CHECKLIST_NAMES.ga4Properties,
+      description: "Service account doesn't have access to any GA4 properties",
+      disabled: false,
+      checklistUrl: {
+        title: 'Grant access',
+        url: 'https://analytics.google.com/analytics/web/',
+      },
+    },
+    firstTimeSetupNotEnabled: {
+      icon: getClockIcon(
+        `You'll need to grant viewer access to your service account in a Google Analytics 4 property. This check will run once the ${CHECKLIST_NAMES.adminApi} has been enabled.`
+      ),
       title: CHECKLIST_NAMES.ga4Properties,
       description: `Enable Analytics Admin API to run this check`,
       disabled: true,
     },
     invalidServiceAccount: {
       icon: getClockIcon(
-        'Service account errors detected, please install a correctly configured service account installation'
+        'This check will not run until a valid service account key has been provided'
```

Comment:
man we should really store this as a variable. way too much repeated use of this to keep copy/pasting it everywhere i think